tests/app/celery/test_monthly_tasks.py

```diff
@@ -0,0 +1,56 @@
+from itsdangerous import URLSafeSerializer
+
+from app.celery.monthly_tasks import (
+    resign_api_keys_task,
+    resign_inbound_sms_task,
+    resign_service_callbacks_task,
+)
+from app.models import ApiKey, InboundSms, ServiceCallbackApi
+from tests.app.db import create_api_key, create_inbound_sms, create_service_callback_api
+
+
+class TestResigning:
+    def test_resign_callbacks(self, sample_service):
+        from app import signer_bearer_token
+
+        signer_bearer_token.serializer = URLSafeSerializer(["k1", "k2"])
+        initial_callback = create_service_callback_api(service=sample_service)
+        bearer_token = initial_callback.bearer_token
+        _bearer_token = initial_callback._bearer_token
```

Comment:
Hmm I don't know how the setter and getter works, but seems like invoking the resign will change the internal value, right?

Reply:
`bearer_token = initial_callback.bearer_token` will call the getter, which will verify the `_bearer_token` and return the unsigned (ie verified) value. 
`_bearer_token = initial_callback._bearer_token` will get the value of `_bearer_token` directly, ie no getter called.

Whe we do something like `callback.bearer_token = new_token` then the setter is called, `new_token` is signed and the signed value is stuck in  `_bearer_token`